docs/zh/deployment/distributed-deployment-guide.md

```diff
@@ -1252,6 +1246,13 @@ portal上“帮助”链接的地址，默认是Apollo github的wiki首页，可
 }
 ```
 
+### 3.1.13 searchByItem.switch - 控制台搜索框是否支持按配置项搜索
```

Comment:
It looks like the [en guide](https://github.com/apolloconfig/apollo/blob/master/docs/en/deployment/distributed-deployment-guide.md#3112-admin-servicesaccesstokens---set-the-access-token-required-by-apollo-portal-to-access-the-apollo-adminservice-for-each-environment) also needs to be fixed.